conf/app.routes

```diff
@@ -8,4 +8,10 @@ GET        /activeCesaRelationship/nino/:nino/saAgentReference/:saAgentReference
 
 GET        /acceptableNumberOfClients/service/IR-PAYE                                   @uk.gov.hmrc.agentassurance.controllers.AgentAssuranceController.acceptableNumberOfPAYEClients
 
-GET        /acceptableNumberOfClients/service/IR-SA                                     @uk.gov.hmrc.agentassurance.controllers.AgentAssuranceController.acceptableNumberOfIRSAClients
\ No newline at end of file
+GET        /acceptableNumberOfClients/service/IR-SA                                     @uk.gov.hmrc.agentassurance.controllers.AgentAssuranceController.acceptableNumberOfIRSAClients
+
+
+GET        /properties/:key                                                             @uk.gov.hmrc.agentassurance.controllers.PropertiesController.getProperty(key: String)
```

Comment:
what about making it more specific, not generic, we will deal with two separate concepts: refusal2dealWith and manually assured

Reply:
I was struggling to come up with a good url (I mentioned it at the stand up). I will come up with something better

Reply:
I would imagine here 2 sets of endpoints, one for R2DWL, another one for manually assured, WDYT?

Reply:
👌 and both sets of endpoints will do the exact same thing. We can go with this